src/components/CatalogMenuItemMobile/index.module.scss

```diff
@@ -0,0 +1,62 @@
+@import 'styles/constants';
+
+.wrapper {
+  display: flex;
+  padding: 15px 32px;
+  margin-top: 15px;
+  background-color: $pageBackground;
+}
+
+.category {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.link,
+.clothing,
+.promotion {
+  font-size: 24px;
+  font-weight: 400;
```

Comment:
are needed because they are interrupted by the styles of other components
![11](https://github.com/Vadim-Kostetskyi/Online-store/assets/113288447/beca8111-253a-4b59-a171-13d4495673d0)